src/Stripe.net/Infrastructure/StringOrObject.cs

```diff
@@ -10,7 +10,7 @@ public static void Map(object value, Action<string> updateId, Action<T> updateOb
         {
             if (value is JObject)
             {
-                T item = ((JToken)value).ToObject<T>();
+                T item = Mapper<T>.MapFromJson(value.ToString());
```

Comment:
Can you document this line explicitly as to what it does, why the change and maybe a TODO to figure out in the future how to avoid the parsed => toString => parsed?